### DISCUSSION TITLE
Thoughts on one of these logos?

### DISCUSSION THREAD

**Discussion by christoph-codes** [christoph-codes/bulb#5] [Q&A] [answered]
Hey @fireprone 

## Logo 1
![Screen Shot 2021-11-12 at 10 56 06 PM](https://user-images.githubusercontent.com/54452490/141609347-a68b71d1-8733-42aa-bf1a-dffdd96bb01d.png)

## Logo 2
![Screen Shot 2021-11-12 at 10 56 01 PM](https://user-images.githubusercontent.com/54452490/141609365-e258b292-3b8d-4251-9102-92b4a97397f2.png)

After picking one of these I will move to color and then we will pick again and then jump back into planning the real stuff! Let me know what you think!

**Comment by fireprone** [comment, marked as answer]
Loving the second option! That's my vote 👍

**Reply by christoph-codes** [reply]
@fireprone Color?
Thoughts behind the colors:
 - Blue: The color of being trustworthy
 - Orange: The color for creativity but also can represent something that is heating up/cooking (idea cooking)
 - Yellow: The color for optimism & enthusiasm. Just so happens to be the color associated with a light bulb 👍  (Felt fitting but not the primary color since that would be the obvious choice)

### Full Color
![Screen Shot 2021-11-15 at 11 36 06 PM](https://user-images.githubusercontent.com/54452490/141941501-33a18c83-2fdb-48da-886e-d4e1830b49a6.png)

### Primary Favicon
![Screen Shot 2021-11-15 at 11 36 12 PM](https://user-images.githubusercontent.com/54452490/141941491-bdec830f-9371-4581-99b9-1ad31b02a541.png)

### Secondary Favicon (One Color)
![Screen Shot 2021-11-15 at 11 36 17 PM](https://user-images.githubusercontent.com/54452490/141941482-3d6f549d-e29f-4b20-97e9-ca75f1936c05.png)

**Reply by fireprone** [reply]
I'm a big fan of the full color! I'd like to explore some darker colors for the primary favicon as the yellow is hard to see against white, but other than that looks great 👍

**Reply by christoph-codes** [reply]
@fireprone That's a good point. I think we can definitely darken the blue we use so yellow shows more. Or we can make a rule that we always show it as the inverse to where we never see the hard to see version... I'm good either way!
